native/src/assets/index.ts

```diff
@@ -40,6 +43,9 @@ export {
   EventThumbnailPlaceholder2,
   EventThumbnailPlaceholder3,
   ExternalLinkIcon,
+  GPSFixed,
+  GPSNotFixed,
+  GPSOff,
```

Comment:
So far we never use this naming afaik, how about
```suggestion
  LocationFixedIcon,
  LocationUnfixedIcon,
  LocationDisabledIcon
```
At least it should all be called `Icon`.